src/content/docs/resources/educational.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Community Resources
+---
+
+import ResourceCards from '~/components/ResourceCards.astro'
+
+See guided examples of adding features to your Astro project, submitted by our community members!
+
+[Astro recipes](https://docs.astro.build/en/recipes/) are typically short, focused how-to guides that walk a reader through completing a working example of a specific task. Recipes are a great way to add new features or behavior to your Astro project by following step-by-step instructions!
+
+Other guides might explain concepts related to an area of content, such as using images or working with MDX.
+
+:::tip[Add your own!]
```

Comment:
Just reminding that this is a placeholder and you'll eventually want your own process here. If you're not having one big `.mdx` page, then people won't simply be editing this page an adding a link.

Have you thought about whether you want people to submit via a form and *you* create the page for them, or whether you will want people to e.g. "create a new `.mdx` file under `src/content/resources/`"?